Tests/MapboxMapsTests/Snapshot/Snapshot.xcassets/testSnapshotAttribution-200.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
```

Comment:
are these for testing purposes only? If so, we can add just a single file outside of the asset catalog to avoid some bloating of files

Reply:
see this [PR](https://github.com/mapbox/mapbox-maps-ios/pull/553/files) for an example